src/server/rpc/procedures/iex-trading/iex-trading.js

```diff
@@ -0,0 +1,130 @@
+// Data attributed to https://iextrading.com/api-exhibit-a
+
+const ApiConsumer = require('../utils/api-consumer');
+
+
+let StockConsumer = new ApiConsumer('iex-trading', 'https://api.iextrading.com/1.0',{cache: {ttl: 3600*24*30*6}});
+
+const queryOpts = {
```

Comment:
This specific object is used in each request. It would be better to either move this into each rpc function call or use `cloneDeep` from lodash.

After looking at it a little more, it looks like you are not actually using most of these fields. It looks like you set the value of `queryString` for each call already and the rest of the values can just be the defaults. Since this is the case, I would just move it into the individual function calls. Something like:

```
this._requestData({queryString: `/stock/${companySymbol}/quote`})
```